server/migrations/20171119180959-create-center.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Centers', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    name: {
+      type: Sequelize.STRING
+    },
+    description: {
+      type: Sequelize.TEXT
+    },
+    location: {
+      type: Sequelize.STRING
+    },
+    address: {
+      type: Sequelize.STRING
+    },
+    capacity: {
+      type: Sequelize.STRING
+    },
+    createdAt: {
+      allowNull: false,
+      type: Sequelize.DATE
+    },
```

Comment:
please do not push migrations files

Reply:
Thank you for the feedback. I will remove it